src/pages/ReportSettingsPage.js

```diff
@@ -84,12 +98,62 @@ class ReportSettingsPage extends Component {
 
         this.state = {
             newRoomName: this.props.report.reportName,
-            error: '',
+            errors: {},
         };
+
+        this.validateAndRenameReport = this.validateAndRenameReport.bind(this);
+    }
+
+    validateAndRenameReport() {
+        if (this.props.report.reportName === this.state.newRoomName) {
+            Growl.success(this.props.translate('newRoomPage.policyRoomRenamed'));
+            return;
+        }
+        if (!this.validate()) {
+            return;
+        }
```

Comment:
```suggestion
        if (this.props.report.reportName === this.state.newRoomName) {
            Growl.success(this.props.translate('newRoomPage.policyRoomRenamed'));
            return;
        }
        if (!this.validate()) {
            return;
        }
```

Please move the validate block above the growl one.